pkg/ocm/helpers.go

```diff
@@ -493,16 +499,16 @@ func GetVersionMinor(ver string) string {
 	return fmt.Sprintf("%d.%d", segments[0], segments[1])
 }
 
-func CheckSupportedVersion(clusterVersion string, operatorVersion string) (bool, error) {
-	v1, err := semver.NewVersion(clusterVersion)
+func IsGreaterThanOrEqual(version1, version2 string) (bool, error) {
```

Comment:
Is the name change necessary here? Dont have a strong opinion on it, but if we are keeping it, can we name the params something different to just be clear on what we are checking is greater than or equal too. 
Same if reverting back to `CheckSupportedVersion` naming, it be good to update the param names to be a bit clearer as its not just checking operator versions.

Reply:
I see that there are multiple references to `CheckSupportedVersion`, were you able to check all the other paths with this change?

Reply:
> I see that there are multiple references to `CheckSupportedVersion`, were you able to check all the other paths with this change?

What do you want me to check exactly? Thanks!

Reply:
If we change the behavior of  `CheckSupportedVersion`, we need to validate it's not affecting other flows that call this function.

Reply:
I'm not changing the behavior, I just changed the signature.

Reply:
I do see a call to `getVersionRawID` that wasn't there before https://github.com/openshift/rosa/pull/1203/files#diff-97477d2275d7797e234adfc6164c02beabfdbdb0eb516faed787f52d1cfb30c1R503.

Reply:
Right, my bad I'll revert that.